.github/workflows/build_release_assets.yml

```diff
@@ -26,11 +26,11 @@ jobs:
 
       - name: Install packaging tools
         run: |
-          pip install build
+          pip install --upgrade pdm
```

Comment:
you could use this [setup PDM ](https://github.com/pdm-project/setup-pdm) GitHub Action that PDM itself uses. And PDM versions should be pinned, I think.

Reply:
Should be good now, I have changed to the action where it made sense.